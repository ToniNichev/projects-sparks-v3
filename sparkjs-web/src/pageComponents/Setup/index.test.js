import React from 'react';
import { shallow, mount } from 'enzyme';
import  { Setup } from '.';
import toJson from 'enzyme-to-json';




describe('Testing Header component', () => {
  it('renders as expected!', () => {
    const wrapper = mount(
      <Setup/>
    );
   expect(toJson(wrapper)).toMatchSnapshot();
  });

  it('test notAvailableModalClose()', () => {

    const loadingValue = true;
    React.useState = jest.fn(initialLoadingValue => [
      loadingValue,
      () => {}
    ]);

    const wrapper = shallow(<Setup/>);
    
    console.log(wrapper.debug());
    
  });
})