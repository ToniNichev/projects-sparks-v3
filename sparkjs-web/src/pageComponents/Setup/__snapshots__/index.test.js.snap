// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing Header component renders as expected! 1`] = `
<Setup>
  <div>
    <div>
      <h1>
        TEST 123
      </h1>
    </div>
    <p>
      ############
    </p>
    <p>
      Wellcome to the setup page
    </p>
    <p>
      <button
        onClick={[Function]}
      >
        !!!!DROP DATABASE
      </button>
    </p>
    <p>
      <button
        onClick={[Function]}
      >
        CREATE TWO UNREGISTERED HUBS DATABASE
      </button>
    </p>
    <p>
      <button
        onClick={[Function]}
      >
        CREATE DATABASE
      </button>
    </p>
    <p>
      <button
        onClick={[Function]}
      >
        CREATE AND POPULATE DATABASE
      </button>
    </p>
    <button
      onClick={[Function]}
    >
      TEST
    </button>
  </div>
</Setup>
`;
